_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
-msgstr ""
+msgstr "Wenn du ein Mikrofon verwendest, während du ein anderes Instrument spielst, kannst du in deinen Einstellungen ein Stereo-Audioeingangssignal verwenden, bei dem ein Kanal mit dem Instrument und der andere Kanal mit einem Mikrofonsignal verbunden ist. Auf den Mikrofonkanal kann ein optionaler Hall-Effekt eingestellt werden."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:56
 msgid "### Buffer LEDs suddenly going red, outages, jittering, weird sounds?"
-msgstr ""
+msgstr "### Puffer-LEDs leuchten plötzlich rot, Ausfälle, [Jitter](https://de.wikipedia.org/wiki/Jitter), seltsame Geräusche?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:58
 msgid "The CPU in your computer may be having issues. Try not to have anything competing with Jamulus (like Zoom Meetings or Facebook live streams) on your machine. Or at least quit them while you're playing. Prevent things like virus scanners doing scans, or software updates happening, etc. Be aware that the reverb setting in Jamulus also uses more CPU the more reverb you have."
-msgstr ""
+msgstr "Möglicherweise gibt es Probleme mit der CPU deines Computers. Versuche alles was mit Jamulus konkurriert (wie Zoom Meetings oder Facebook-Livestreams) von deinem Computer zu entfernen. Oder beende diese zumindest, während du spielst. Verhindere dass z. B. Virenscanner Scans durchführen oder Software-Updates durchgeführt werden, usw. Beachte dass auch die Hall-Einstellung in Jamulus mehr CPU verbraucht, je mehr Hall eingestellt wird."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:60
 msgid "### Ping times and latency start well, then get worse, causing issues"
-msgstr ""
+msgstr "### Ping-Zeiten und Latenzzeiten beginnen gut, werden dann schlechter und verursachen Probleme"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:62
 msgid "This can indicate something else is competing with Jamulus on your network, so make sure nobody is watching HD Netflix movies or taking part in Zoom video conferences while you are playing. A more permanent solution for technically-minded users may be found by investigating [the issue of buffer bloat](https://www.bufferbloat.net/projects/bloat/wiki/) on their router and whether you can implement Smart Queue Management (SQM). [More details here](https://www.bufferbloat.net/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/)."
-msgstr ""
+msgstr "Dies kann darauf hindeuten, dass ein anderer Dienst in deinem Netzwerk mit Jamulus konkurriert. Stelle also sicher, dass niemand HD-Netflix-Filme ansieht oder an Zoom-Videokonferenzen teilnimmt, während du spielst. Du kannst auch [das Problem der Pufferüberlastung](https://www.bufferbloat.net/projects/bloat/wiki/) auf deinem Router untersuchen und prüfen, ob du Smart Queue Management (SQM) implementieren kannst. [Weitere Einzelheiten findest du hier](https://www.bufferbloat.net/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/)."
```

Comment:
Dies --> Das

Das Problem der Pufferüberlastung --> Du kannst auch ein Problem mit "Bufferbloat" haben. 

(Some rather technical speech might get less understandable if it is translated, I think)

Reply:
Corrected